tests/core/test_grid.py

```diff
@@ -864,6 +864,18 @@ def test_create_image_data_from_specs():
     assert grid != grid_from_grid
 
 
+@pytest.mark.parametrize('dims', [None, (0, 0, 0), (1, 0, 0), (0, 1, 0), (0, 0, 1)])
+def test_image_data_empty_init(dims):
+    image = pv.ImageData(dimensions=dims)
+    assert image.n_points == 0
+    assert image.n_cells == 0
+    assert image.area == 0
+    assert image.volume == 0
+
+    points = image.points
+    assert np.array_equal(points, np.empty((0, 3)))
```

Comment:
The fact that `np.empty` gives zeros here is not guaranteed.  See notes section in [`numpy.empty` docs](https://numpy.org/doc/stable/reference/generated/numpy.empty.html#numpy-empty)

```suggestion
    assert np.array_equal(points, np.zeros((0, 3)))
```

Reply:
See this [comment](https://github.com/pyvista/pyvista/pull/6525#discussion_r1720987894). Seems like it doesn't matter which method is used to create the empty array